executors/cmdexec/volume.go

```diff
@@ -69,6 +69,10 @@ func (s *CmdExecutor) VolumeCreate(host string,
 
 	commands = append(commands, s.createVolumeOptionsCommand(volume)...)
 
+	//Set many bricks per node volume parameters
+    commands = append(commands, fmt.Sprintf("gluster --mode=script volume %v set cluster.brick-multiplex on", volume.Name))
+    commands = append(commands, fmt.Sprintf("gluster --mode=script volume %v set transport.listen-backlog 100", volume.Name))
```

Comment:
Maybe add these option to "VolumeRequest" object in [volume_entry.go#L139](https://github.com/cloud-tools/heketi/blob/ffbd7a61b7ea8c585bd03d3896a8f5a722f5e1d7/apps/glusterfs/volume_entry.go#L139), so that these commands will be automaticaly generated in [volume.go#L175](https://github.com/cloud-tools/heketi/blob/ffbd7a61b7ea8c585bd03d3896a8f5a722f5e1d7/executors/cmdexec/volume.go#L175)